pkg/opensearch/response_parser.go

```diff
@@ -119,8 +190,8 @@ func processRawDataResponse(res *es.SearchResponse, timeField string, queryRes b
 			}
 		}
 
-		if timestamp, ok := getTimestamp(hit, timeField); ok {
-			doc[timeField] = timestamp
+		if timestamp, ok := getTimestamp(hit, configuredFields.TimeField); ok {
```

Comment:
Diff view may look like it changes `processRawData`, but please note the added code is `processLogsResponse`. `processRawData` only uses the configured TimeField